tests/Feature/WebRouteTest.php

```diff
@@ -103,6 +103,8 @@ public function test_results_route_retrieves_mismatches()
                 'id' => $mismatch->id,
                 // Casting values to string, as it seems that the inertia
                 // testing helper also converts all values to strings
+                'item_id' => (string) $mismatch->item_id,
+                'statement_guid' => (string) $mismatch->statement_guid,
```

Comment:
these two are not actually new, they've always been there, right? It's just that now they are needed, and therefore you newly include them in the test?

Reply:
I also noticed this

Reply:
Yes indeed, as long as I don't need them, there is no need to test for them, otherwise tests will become brittle.